candle-examples/examples/reinforcement-learning/README.md

```diff
@@ -2,12 +2,32 @@
 
 Reinforcement Learning examples for candle.
 
+## System wide python
+
 This has been tested with `gymnasium` version `0.29.1`. You can install the
 Python package with:
 ```bash
 pip install "gymnasium[accept-rom-license]"
 ```
 
+## Mac OS uv venv python
+
+```bash
+$ uv venv
+$ source .venv/bin/activate
+$ uv pip install gymnasium
+$ which python3.xx 
+> /path/to/.venv/bin/python3.xx
+$ find ~/.local/share/uv/python -name "libpython3.xx.dylib"
+> /path/to/pythondir/lib/libpython3.xx.dylib
+$ cp /path/to/pythondir/lib/libpython3.xx.dylib ~/libpython3.xx.dylib.backup
+$ install_name_tool -id "/path/to/pythondir/lib/libpython3.xx.dylib" "/path/to/pythondir/lib/libpython3.xx.dylib"
+$ otool -L /path/to/pythondir/lib/libpython3.xx.dylib
```

Comment:
oh wow, why is this necessary? (just curious)

Reply:
I'm honestly not sure. This was a headache for a while. For some reason the rust linker looks for the python libs in `/install/lib` which can't exist because it's in the read only file system on mac. The only way I've found to fix this is to change the install name of the dll to the location where it's supposed to be instead of `/install/lib/`.

Reply:
That's odd, I never had to do something like this and wouldn't expect candle to need anything specific here compared to some standard pyo3 code.
If it's a common issue I would expect pyo3 folks to have encountered it somehow.

Reply:
My guess is that it's a problem with using venv on [UV](https://github.com/astral-sh/uv). It's pretty new and does some different things with symlinks.

Reply:
Some potentially related discussion here: https://github.com/PyO3/pyo3/issues/4813. In particular, [this looks promising](https://github.com/PyO3/pyo3/issues/4813#issuecomment-2617037553).

If an easy config is not possible, I'd maybe suggest to provide simple steps for standard venv and a warning that `uv` is not compatible with `pyo3`.

Reply:
The 

`DYLD_FALLBACK_LIBRARY_PATH=/Users/kb/.local/share/uv/python/cpython-3.12.7-macos-aarch64-none/lib/ CARGO_BUILD_RUSTFLAGS="-L /Users/kb/.local/share/uv/python/cpython-3.12.7-macos-aarch64-none/lib/" cargo run --example reinforcement-learning --features=pyo3 --package candle-examples -- pg` 

was the first thing I tried and I could not get it to work. 

It seems to be a problem with PYTHONHOME and PYTHONPATH with uv.

I'll remove the procedure I used to get it to work and just provide a warning. I can attach it as a comment to this thread for archival purposes.